apis/config.go

```diff
@@ -208,12 +212,7 @@ func SetDownloadDynamicDefaults(cfg *EtcdAdmConfig) error {
 	return setDynamicDefaults(cfg)
 }
 
-// SetDefaults sets configuration values defined at build time
-func SetDefaults(cfg *EtcdAdmConfig) {
-	cfg.UnitFile = constants.UnitFile
-	cfg.EnvironmentFile = constants.EnvironmentFile
-	cfg.EtcdctlEnvFile = constants.EtcdctlEnvFile
```

Comment:
This pattern has evolved because it supports defaults in both flags and when used in code.  IMO we should set the defaults in SetDefaults, and then the flags should reference the values in cfg, but we don't _always_ do that.

But if we're following the pattern, then we would set ClientPort and PeerPort to constants.DefaultClientPort / constants.DefaultPeerPort here